src/ngx_http_lua_accessby.c

```diff
@@ -374,4 +373,20 @@ ngx_http_lua_access_by_chunk(lua_State *L, ngx_http_request_t *r)
     return NGX_DECLINED;
 }
 
+static ngx_int_t
+ngx_http_lua_access_by_chunk(lua_State *L, ngx_http_request_t *r)
```

Comment:
@yangshuxin In that case, it's intentional. Because nginx delegates the decision into the content handler serving that location targeted by the custom error page (internal) redirection. And for ngx_lua, it's still the normal "fall-through" case :)